pushy/src/main/java/com/relayrides/pushy/apns/AuthenticationTokenSupplier.java

```diff
@@ -27,18 +27,23 @@
  */
 class AuthenticationTokenSupplier {
 
-    private final Signature signature;
```

Comment:
I'd really love to find a way to not hang onto this, but `Signature` isn't cloneable, and we definitely want to create a new `AuthenticationTokenSupplier` for each handler.